.github/workflows/ci.yml

```
@@ -7,20 +7,20 @@ jobs:
   unit:
     name: "Unit tests"
     runs-on: ubuntu-20.04
```

Dernière suggestion que je vois sinon  bravo 👏
```suggestion
    runs-on: ubuntu-22.04
```